requirements.txt

```diff
@@ -1,4 +1,5 @@
+graphix>=0.2.8
 numpy>=1.22,<1.26
 qiskit>=0.42
```

Comment:
Could you remove the 'graphix' here to avoid the circular reference issue (we faced that issue before)?
Also, could you temporarily set 'qiskit==0.42' in this PR?